src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
+/* eslint-disable max-len */
 'use strict';
 
-// write your code here
+function getNumbersFromNodes(nodeList) {
+  const numbersArray = [];
+
+  for (const node of nodeList) {
+    const number = +node.textContent.split(',').join('');
+
+    numbersArray.push(number);
+  }
+
+  return numbersArray;
+}
```

Comment:
You could easily count here total using this for-of loop instead of creating additional array. Simplify